polyglot-chat-app/src/main/resources/scripts/plot.R

```diff
@@ -1,3 +1,44 @@
+ #/
```

Comment:
In the R language, you just add # before each line. There is no opening/closing `#/` tag. See https://github.com/graalvm/examples/blob/423444367e03901f04a8f485a31a6c3002f6a76f/fastr_scalar/gameoflife.R for example.